changelog.json

```
@@ -1,5 +1,10 @@
 {
   "releases": {
+    "2.6.3-beta1": [
+      "[Fixed] When fetching a repository, the app will fast-forward all possible branches (except the current branch) - #11387",
+      "[Improved] Enable spellcheck on commit summary and description - #1597. Thanks @Hovemeyer5!",
```

We typically leave our team's contributions out of these notes so I'm going to remove it here 😄 
```suggestion
      "[Improved] Enable spellcheck on commit summary and description - #1597",
```